lms/djangoapps/grades/tests/test_events.py

```diff
@@ -4,16 +4,29 @@
 
 from unittest import mock
 
+from ccx_keys.locator import CCXLocator
 from django.utils.timezone import now
 from openedx_events.learning.data import (
+    CcxCourseData,
+    CcxCoursePassingStatusData,
     CourseData,
-    PersistentCourseGradeData
+    CoursePassingStatusData,
+    PersistentCourseGradeData,
+    UserData,
+    UserPersonalData
+)
+from openedx_events.learning.signals import (
+    CCX_COURSE_PASSING_STATUS_UPDATED,
```

Comment:
I didn't know anyone else was using CCX. I wonder when and if CCX documentation could be improved? I don't know where its docs live. It would be great if there were even a README in lms/djangoapps/ccx.

Reply:
The docs I know of are here: https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/set_up_course/studio_add_course_information/custom_courses.html

Reply:
There are two major operators of Open edX in the community that are heavily relying on CCX courses to these days, Pearson and Campus. As per my knowledge, all the CCX functionality is still operable and compatible with learning MFE.